core/src/main/scala/codesearch/core/search/Search.scala

```diff
@@ -40,14 +40,11 @@ trait Search {
     * @param relativePath path to source code
     * @return package name and url to repository
     */
-  def packageName(relativePath: String): Option[Package] = {
-    val slashes = if (relativePath.startsWith("./")) 4 else 3
-    relativePath.split('/').drop(slashes).toList match {
-      case libName :: version :: _ =>
-        val decodedName = URLDecoder.decode(libName, "UTF-8")
-        Some(Package(s"$decodedName-$version", buildRepUrl(decodedName, version)))
-      case _ => None
-    }
+  def packageName(relativePath: String): Option[Package] = relativePath.split('/').drop(3).toList match {
+    case libName :: version :: _ =>
+      val decodedName = URLDecoder.decode(libName, "UTF-8")
+      Some(Package(s"$decodedName-$version", buildRepUrl(decodedName, version)))
+    case _ => None
```

Comment:
Please explain what does mean this changes. Also, you can leave comments in the code.

Reply:
I want you to explain this changes.
Was:
```scala
val slashes = if (relativePath.startsWith("./")) 4 else 3
relativePath.split('/').drop(slashes)
```
Now:
```scala
relativePath.split('/').drop(3)
```